src/memory_map.hpp

```diff
@@ -21,8 +21,136 @@ enum addresses : uint8_t {
     MPUREG_I2C_SLV0_DO = 0x63,
     MPUREG_I2C_SLV0_REG = 0x26,
     MPUREG_EXT_SENS_DATA = 0x49,
+    READ_FLAG = 0x80,
```

Comment:
This should be an enum class, not an enum.

Reply:
Done!